core-services/image-mirroring/openshift/mapping_origin_4_10

```diff
@@ -34,6 +34,7 @@ registry.ci.openshift.org/origin/4.10:cli-artifacts quay.io/openshift/origin-cli
 registry.ci.openshift.org/origin/4.10:cloud-credential-operator quay.io/openshift/origin-cloud-credential-operator:4.10 quay.io/openshift/origin-cloud-credential-operator:4.10.0 quay.io/openshift/origin-cloud-credential-operator:latest
 registry.ci.openshift.org/origin/4.10:cloud-event-proxy quay.io/openshift/origin-cloud-event-proxy:4.10 quay.io/openshift/origin-cloud-event-proxy:4.10.0 quay.io/openshift/origin-cloud-event-proxy:latest
 registry.ci.openshift.org/origin/4.10:cloud-network-config-controller quay.io/openshift/origin-cloud-network-config-controller:4.10 quay.io/openshift/origin-cloud-network-config-controller:4.10.0 quay.io/openshift/origin-cloud-network-config-controller:latest
+registry.ci.openshift.org/origin/4.10:cluster-api quay.io/openshift/origin-cluster-api:4.10 quay.io/openshift/origin-cluster-api:4.10.0 quay.io/openshift/origin-cluster-api:latest
```

Comment:
What is in the cluster-api image?

Reply:
This is a fork of upstream cluster-api project https://github.com/openshift/cluster-api, it will be used by hypershift and cloud team

Reply:
So this is a controller?  An operator?  What use will this image be put to inside of openshift?

Reply:
Ok, should be named cluster-capi-controllers to be consistent with the operator.

Reply:
done